src/edu/csus/ecs/pc2/clics/API202306/CLICSLocation.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * CLICS geographic location
+ * Contains information the location of another object
+ * 
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CLICSLocation {
+    private static final double MAX_LATITUDE = 90.0;
+    private static final double MAX_LONGITUDE = 180.0;
```

Comment:
Since latitude and longitude range from [-90..90] and [-180..80] (respectively), it seems like there should also be constants
```
private static final double MIN_LATITUDE = -90.0;
private static final double MIN_LONGITUDE = -180.0;
```
(See below for the place where these would be used...)